test/everypolitician/popolo/event_test.rb

```diff
@@ -1,13 +1,12 @@
 require 'test_helper'
-
 class EventTest < Minitest::Test
-  def test_reading_popolo_events
-    popolo = Everypolitician::Popolo::JSON.new(
-      events: [{ id: 'term/8', name: '8th Verkhovna Rada', start_date: '2014-11-27' }]
-    )
-    event = popolo.events.first
+  def events
+    Everypolitician::Popolo.read('test/fixtures/turkey-ep-popolo-v1.0.json').events
```

Comment:
I think you'll want to memoize that! That's a pretty big file to keep reading every time.